src/tools/ApiChecker/ScaffoldTest.cs

```diff
@@ -135,5 +173,21 @@ private static void AddTest(CodeCompileUnit @class, string testDescribe, string
                         new CodePrimitiveExpression("This test is not yet implemented.")),
                 }));
         }
+
+        /// <summary>
+        /// Describes the options for scaffolding the tests.
+        /// </summary>
+        [Verb("scaffold-test", HelpText = "Takes a spec.ts file and scaffolds the C# test.")]
+        internal class ScaffoldTestOptions
```

Comment:
Any reason to move the class here?

Reply:
So, they are completely tied to each other and it makes no sense to have them anywhere outside. While it seems a bit weird to have a nested class, I don't see any good reason to have them as separate files - i.e. for code readability, this makes it easier, for portability (when we move this piece somewhere else) it will make it easier too, etc. :-)

Reply:
So, are you OK if I resolve this? :-)